src/uwtools/config/atparse_to_jinja2.py

```diff
@@ -6,30 +6,36 @@
 import sys
 from typing import IO, Optional
 
+from uwtools.utils.file import readable
 
-def convert(input_template: str, outfile: Optional[str] = None, dry_run: bool = False) -> None:
+
+def convert(
+    input_file: Optional[str] = None, output_file: Optional[str] = None, dry_run: bool = False
+) -> None:
     """
-    Renders a Jinja2 template using user-supplied configuration options via YAML or environment
-    variables.
+    Replaces atparse @[] tokens with Jinja2 {{}} equivalents.
+
+    If no input file is given, stdin is used. If no output file is given, stdout is used. In dry-run
+    mode, output is written to stderr.
 
-    :param input_template: Path to the template containing atparse syntax.
-    :param outfile: The file to write the converted template to.
+    :param input_file: Path to the template containing atparse syntax.
+    :param output_file: Path to the file to write the converted template to.
     :param dry_run: Run in dry-run mode?
     :raises: RuntimeError if neither an output file or dry-run are specified.
     """
 
     def write(f_out: IO) -> None:
-        with open(input_template, "r", encoding="utf-8") as f_in:
+        with readable(input_file) as f_in:
```

Comment:
Once merged, should we in the future be replacing 
`with open(...., "r", .....) as f:` with `uwtools.utils.file.readable()` ?

Reply:
Good question. I think it's safe to call if you can be sure that you're passing in a `str` or `Path` object, or when you're ok with getting `stdin` if you, for whatever reason, have a falsey path value. For example, blindly passing in a value obtained from calling `.get()` on a `dict`, which returns `None` if the key doesn't exist, could be dangerous, as reading from `stdin` will hang if nothing has been piped into it. Probably the edge case where this could be a problem will be rare, but it's something to ask yourself before calling this function. Using it certainly eliminates a bit of repetitive boilerplate.